nextmv/nextmv/cloud/client.py

```diff
@@ -397,7 +397,7 @@ def upload_to_presigned_url(
 
         if upload_data is not None:
             kwargs["data"] = upload_data
-        elif tar_file is not None:
+        elif tar_file is not None and tar_file != "":
```

Comment:
```suggestion
        elif tar_file:
```

Isn't it equivalent to this?

Almost I suppose. Actually, potentially better to be more explicit. :relaxed: 
So, feel free to leave it like it is.

Reply:
Yes I prefer to be explicit... I'll leave it as is...